api/app/settings/production.py

```diff
@@ -1,8 +1,3 @@
 from app.settings.common import *
 
 REST_FRAMEWORK["PAGE_SIZE"] = 999
-
-# Needed by Elastic Beanstalk to correctly identify incoming protocol
-# https://docs.aws.amazon.com/elasticloadbalancing/latest/application/x-forwarded-headers.html#x-forwarded-proto
-# Although in our case SSL termination happens in Cloudfront so the header is named as below.
-SECURE_PROXY_SSL_HEADER = ("HTTP_CLOUDFRONT_FORWARDED_PROTO", "https")
```

Comment:
This has been removed? How are we going to configure SaaS for this?

Reply:
The env var.

Reply:
`SECURE_PROXY_SSL_HEADER_NAME=HTTP_CLOUDFRONT_FORWARDED_PROTO`